e2e/testconfig/testconfig.go

```diff
@@ -111,24 +192,42 @@ func FromEnv() TestConfig {
 	}
 
 	return TestConfig{
-		ChainAConfig: ChainConfig{
-			Image:  chainAImage,
-			Tag:    chainATag,
-			Binary: chainBinary,
+		ChainConfigs: []ChainConfig{
+			{
+				Image:  chainAImage,
+				Tag:    chainATag,
+				Binary: chainBinary,
+			},
+			{
+				Image:  chainBImage,
+				Tag:    chainBTag,
+				Binary: chainBinary,
+			},
 		},
-		ChainBConfig: ChainConfig{
-			Image:  chainBImage,
-			Tag:    chainBTag,
-			Binary: chainBinary,
+		UpgradeConfig: UpgradeConfig{
+			PlanName: upgradePlan,
```

Comment:
I still need to add this to the json file, but I was actually thinking we probably want some sort of mapping of tag to plan name. This way we can just create a config file once, then be able to run upgrade tests whenever we want without specifying plan name manually based on each test. E.g.

```json
{
   "v7.0.0": "v7",
   "v4.3.0": "normal upgrade" 
}
```

WDYT @damiannolan ?